client/src/components/Login/SignUpForm/index.js

```diff
@@ -32,11 +32,13 @@ const ButtonWrapper = styled.div`
   padding: ${spacing8} 0;
 `;
 
-const SignUpForm = ({ switchView }) => {
+const SignUpForm = ({ updateAuthToken, switchView, redirectToHome}) => {
```

Comment:
You will also need to pass the updateAuthToken and redirectToHome from the SignUpView. (Look at LoginView if you need help).